app/changelog/chart-operator.yaml

```diff
@@ -1,3 +1,13 @@
+- version: 0.12.0
+  description: Improvements to release and status resources. 
+  kind: changed
+  urls:
+    - https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0120-2020-03-09
```

Comment:
This is a minor release with quite a few changes. So this links to the chart-operator changelog.